CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v1.0.0-beta1 (20 Feb 2017)
```

Comment:
After recent issues (e.g. me trying to fix cherami yesterday) I think having a tag to point at (instead of "master", which glide seems to sometimes give you an old version) will be much more useful. Otherwise I worry that people will say `version: master` but really be on a pre-beta commit. Essentially this is to reduce support burden, so we can ensure everybody's on the beta we expect them to be using.